dev-php/xdebug-client/xdebug-client-2.6.0_alpha1.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+MY_PV="${PV/_/}"
+MY_PV="${MY_PV/rc/RC}"
+
+DESCRIPTION="Xdebug client for the Common Debugger Protocol (DBGP)"
+HOMEPAGE="http://www.xdebug.org/"
+SRC_URI="http://pecl.php.net/get/xdebug-${MY_PV}.tgz"
+
+LICENSE="Xdebug"
+SLOT="0"
+KEYWORDS="~amd64 ~hppa ~ppc ~ppc64 ~x86"
+IUSE="libedit"
+
+S="${WORKDIR}/xdebug-${MY_PV}/debugclient"
```

Comment:
in the other ebuild you did it correctly and put it after `RDEPEND`, please do it here too. Beyond that, looks good

Reply:
done